Merge changes from -dev

diabelko opened the pull request.

diabelko committed:
Add dmesg publishing to logserver

diabelko committed:
Gather last 10 commit info for each repository

kklimonda committed:
Move Juniper/contrail-common to gerrit source

diabelko committed:
Compress unittest log output before sending to logserver

kklimonda commented on roles/contrail-upload-dmesg/tasks/main.yaml:
```diff
@@ -0,0 +1,15 @@
+  register: dmesg
+  no_log: true
+
+- name: Save dmesg output to a file
```
It can probably be replaced with:
```ansible
- name: Write host dmesg output
  delegate_to: localhost
  copy:
    content: "{{ dmesg.stdout }}"
    dest: "{{ zuul.executor.log_root }}/zuul-info"
```

diabelko replied:
Fixed.

kklimonda reviewed (approved):
Either rewrite one of the patches, or create a follow-up PR with the fix for dmesg.

diabelko committed:
Clean up sending dmesg and git log

  Copying a single file locally is unnecessary, it can be done directly
  on the executor.
